src/browzine-summon-adapter.js

```diff
@@ -124,30 +131,63 @@ angular.module("summonApp.directives")
     return template;
   };
 
-  return {
-    link: function(scope, element, attributes) {
-      if(!shouldEnhance(scope)) {
-        return;
+  function documentSummary(scope, element) {
+    if(!shouldEnhance(scope)) {
+      return;
+    }
+
+    var endpoint = getEndpoint(scope);
+
+    $.getJSON(endpoint, function(response) {
```

Comment:
Is this less of an "angular-y" way of doing things, or still perfectly valid?  

Does an angular app pretty much always have jQuery installed?

Reply:
Yep, Angular is dependent on jQuery, so jQuery is always installed. The whole reason for the rewrite is to decouple from Angular so the BrowZine Summon Adapter runs 100% of the time and we eliminate the risk of Summon never fixing their async LABjs use to load 3rd-Party Angular extensions correctly - i.e. Before starting the manual Angular bootstrap. My point is this update makes this a JavaScript app that tries to read Angular data.

It doesn't run on the Angular framework at this point. We could switch back in the future, of course, but Summon would have to fix that LABjs issue and wait until all scripts are loaded before starting the Angular bootstrap.

Reply:
Thanks for the clarification!